documentation/server/configuration.md

```diff
@@ -9,12 +9,12 @@ description: Learn more about Configuration
 We create a very small and simple to use configuration service. This service would be used inside the framework, but if you want, you can use it too.
 
 {% hint style="info" %}
-We are not do any fancy magic stuff here. Thanks goes [`Lodash`](https://lodash.com/)
+We are not do any fancy magic stuff here. Thanks go to [`Lodash`](https://lodash.com/)
```

Comment:
`We are not do any fancy magic stuff here.` to We are not doing any fancy magic stuff here.